### PR TITLE
fix(build): fix the travis-ci build to deploy with assets

### PR DESCRIPTION
**Pull request by jcreamer898**
Fixes the travis ci build to make sure the compiled assets are included in the npm package.

### CODE REVIEW

**Comment by benmvp** [comment]
While you're here, can you remove all the extra junk in the package too?

**Comment by jcreamer898** [comment]
Yessir.

File: .npmignore [hunk 1]
```diff
@@ -1,21 +0,0 @@
-# Configuration files
```
Comment (benmvp) [inline .npmignore]: :rip:

**Pull request merged**